### PR TITLE
Update rds_vdi-recommendations-1909.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Updated quote formats. 
Updated script names to align with 1909 version of The VDI Guys file names.
Tiny rewrite of PowerShell script to kill running tasks.
Updated optimization settings to default user profile registry hive with settings from The VDI Guys version 1909.

### CODE REVIEW

File: WindowsServerDocs/remote/remote-desktop-services/rds_vdi-recommendations-1909.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ You might note that this document and the associated scripts on GitHub do not mo
 
 ### Universal Windows Platform (UWP) application cleanup
 
-One of the goals of a VDI image is to be as light as possible. One way to reduce the size of the image is to remove UWP applications that won't be used in the environment. With UWP apps, there are the main application files, also known as the payload. There is a small amount of data stored in each user’s profile for application specific settings. There is also a small amount of data in the ‘All User’s’ profile.
+One of the goals of a VDI image is to be as light as possible. One way to reduce the size of the image is to remove UWP applications that won't be used in the environment. With UWP apps, there are the main application files, also known as the payload. There is a small amount of data stored in each user's profile for application specific settings. There is also a small amount of data in the 'All User's' profile.
```
Comment (illfated) [inline WindowsServerDocs/remote/remote-desktop-services/rds_vdi-recommendations-1909.md:143]: I am fairly sure that the form "All User" (singular) is not a useful grammatical form (maybe even invalid), so we can skip the possessive singular form here:
```suggestion
One of the goals of a VDI image is to be as light as possible. One way to reduce the size of the image is to remove UWP applications that won't be used in the environment. With UWP apps, there are the main application files, also known as the payload. There is a small amount of data stored in each user's profile for application specific settings. There is also a small amount of data in the 'All Users' profile.
```
Even though it is possible to use the possessive form of "All Users" as in "the All Users' profile", it is rarely needed in technical documentation.

**Comment by illfated** [comment]
Other than that, I appreciate the codestyle improvement of correcting the curly quotes to straight quotes (single and double quotation marks).
Server scripting is not my strong side, so I will avoid reviewing that part.

File: WindowsServerDocs/remote/remote-desktop-services/rds_vdi-recommendations-1909.md (hunk 1 above)
Comment (baardhermansen) [reply WindowsServerDocs/remote/remote-desktop-services/rds_vdi-recommendations-1909.md]: Indeed, you are correct. I didn't even notice it. The name of the folder is "All Users", so there should never have been an apostrophe there to begin with.

**Comment by illfated** [comment]
#sign-off

**Comment by eross-msft** [comment]
@baardhermansen  This is in review by the product team. As soon as they sign off, I will update this pull request. Thanks!!

**Comment by illfated** [comment]
@baardhermansen : 5 commits have been made to the document since you opened this PR (see also history for this document: https://github.com/MicrosoftDocs/windowsserverdocs/commits/master/WindowsServerDocs/remote/remote-desktop-services/rds_vdi-recommendations-1909.md).

Will you be able to resolve the file conflict in this PR?

**Comment by baardhermansen** [comment]
@illfated I believe i've successfully resolved the conflicts.

**Comment by illfated** [comment]
#sign-off

**Comment by eross-msft** [comment]
@illfated @baardhermansen  Thank you for all of your efforts with this. Your PR kicked off a conversation in the product group, but I'm not sure where it landed. 

@Heidilohr Did this pull request get rolled into the updates you are making with the product group? Thanks!

**Comment by baardhermansen** [comment]
> @illfated @baardhermansen Thank you for all of your efforts with this. Your PR kicked off a conversation in the product group, but I'm not sure where it landed.

Heh, that's not necessarily a good sign :P 
Alright, i guess there'll be an update eventually.

**Comment by eross-msft** [comment]
@illfated @baardhermansen It looks like they plan to merge all of these guides into one "super" guide, but not for a while yet. So I'm going to go ahead and merge these changes now. Then, your updates will just become a part of the new world when the merge happens down the road.

**Pull request merged**